model_analyzer/reports/report_manager.py

```diff
@@ -945,7 +945,7 @@ def _build_constraint_strings(self) -> Dict[str, str]:
                 self._config).items():
             strs = []
             if model_constraints:
-                for metric, constraint in model_constraints.items():
+                for metric, constraint in model_constraints.to_dict().items():
                     metric_header = MetricsManager.get_metric_types(
                         [metric])[0].header(aggregation_tag='')
                     for constraint_type, constraint_val in constraint.items():
```

Comment:
Where is the constraint class? I thought we talked about creating this? I would have expected this to be a method call

Reply:
Added `items()` support for the ModelConstraints class